rotkehlchen/tests/api/test_snapshots.py

```diff
@@ -85,35 +86,35 @@ def _create_snapshot_with_valid_data(directory: str, timestamp: Timestamp) -> No
                 'category': 'asset',
                 'asset_identifier': 'BTC',
                 'amount': '1',
-                'eur_value': '40000.000',
+                'usd_value': '40000.000',
             },
         )
 
-    path = Path(directory) / LOCATION_DATA_FILENAME
+    path = Path(directory) / LOCATION_DATA_IMPORT_FILENAME
     with open(path, 'w') as f:
-        fieldnames = ['timestamp', 'location', 'eur_value']
+        fieldnames = ['timestamp', 'location', 'usd_value']
         writer = csv.DictWriter(f, fieldnames=fieldnames)
         writer.writeheader()
         writer.writerow(
             {
-                'timestamp': timestamp_to_date(timestamp, '%Y-%m-%d %H:%M:%S'),
+                'timestamp': timestamp,
                 'location': 'blockchain',
-                'eur_value': '100.555',
+                'usd_value': '100.555',
             },
         )
         writer.writerow(
             {
-                'timestamp': timestamp_to_date(timestamp, '%Y-%m-%d %H:%M:%S'),
+                'timestamp': timestamp,
                 'location': 'total',
-                'eur_value': '41000.555',
+                'usd_value': '41000.555',
             },
         )
 
 
 def _create_snapshot_with_valid_data_for_post(directory: str, timestamp: Timestamp) -> None:
     path = Path(directory) / BALANCES_FOR_IMPORT_FILENAME
     with open(path, 'w') as f:
-        fieldnames = ['timestamp', 'category', 'asset_identifier', 'amount', 'eur_value']
+        fieldnames = ['timestamp', 'category', 'asset_identifier', 'amount', 'usd_value']
```

Comment:
you should:

1. Abstract some of the stuff you have here so that the code does not repeat itself as you do that a lot in this file
2. Test both types of snapshots. Both the one that is for human consumption and the one that is supposed to be for importing (machine consumption)